examples/common/pigweed/rpc_services/Device.h

```diff
@@ -418,6 +419,18 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         {
             mCommissionableDataProvider.SetSpake2pVerifier(ByteSpan(request.verifier.bytes, request.verifier.size));
         }
+
+        // Resetart commissioning window to recache the value:
+        {
+            DeviceLayer::StackLock lock;
+            Server::GetInstance().GetCommissioningWindowManager().CloseCommissioningWindow();
+        }
+        // Let other tasks possibly work since Commissioning window close/open are "heavy"
+        {
+            DeviceLayer::StackLock lock;
+            Server::GetInstance().GetCommissioningWindowManager().OpenBasicCommissioningWindow();
```

Comment:
What if the commissioning window wasn't already open?

Reply:
Done